src/features/basic/rename-tab.ts

```diff
@@ -0,0 +1,14 @@
+function updateTabTitle() {
+  const screenName = document
+    .querySelector(`.${C.ScreenControls.currentScreenName}`)
+    ?.textContent?.trim();
+  document.title = screenName ? `${screenName} - Prosperous Universe` : 'Prosperous Universe';
+}
+
+function init() {
+  // Listen for URL changes using the hashchange event
+  window.addEventListener('hashchange', updateTabTitle);
+  updateTabTitle();
```

Comment:
hashchange won't be needed once you implement the changes above. Inline `updateTabTitle` into `init`.